install/update_93_94.php

```diff
@@ -290,6 +290,19 @@ function update93to94() {
    $migration->addField("glpi_changevalidations", "timeline_position", "tinyint(1) NOT NULL DEFAULT '0'");
    $migration->addField("glpi_problemtasks", "timeline_position", "tinyint(1) NOT NULL DEFAULT '0'");
 
+   /** Give all existing profiles access to personalizations for legacy functionality */
```

Comment:
The whole piece of code can be replaced by
```suggestion
   /** Give all existing profiles access to personalizations for legacy functionality */
   $migration->addRight('personalization', READ | UPDATE);
```